components/tooltip/tooltip.js

```diff
@@ -351,6 +351,14 @@ class Tooltip extends RtlMixin(LitElement) {
 				padding-right: 1rem;
 			}
 
+			::slotted(p:first-child) {
+				margin-top: 0 !important;
+			}
+
+			::slotted(p:last-child) {
+				margin-bottom: 0 !important;
```

Comment:
Looks like these changes are for all tooltips, not just the new `d2l-help-tooltip` - is there any risk of this negatively affecting existing users of the tooltip?

Reply:
Yup, my understanding was that this was something that we wanted to change for the Tooltip in general, as per [here](https://d2l.slack.com/archives/C03FC0GDVB7/p1658338548469239?thread_ts=1658257409.018289&cid=C03FC0GDVB7).

However, theoretically if other consumers/teams have vdiff tests of their own, specifically with paragraphs slotted in, then they may notice those tests failing. Otherwise there shouldn't be any other risk.